WinPort/src/sudo/sudo_askpass_ipc.cpp

```diff
@@ -1,3 +1,7 @@
+#ifdef __HAIKU__
+#include <cstring>
+#endif
```

Comment:
no need to #ifdef __HAIKU__ here, also use #include <string.h> just for symmetry - as its used everywhere else like string.h